Meter.c

```diff
@@ -383,7 +383,16 @@ static void GraphMeterMode_draw(Meter* this, int x, int y, int w) {
    gettimeofday(&now, NULL);
    if (!timercmp(&now, &(data->time), <)) {
       struct timeval delay = { .tv_sec = (int)(CRT_delay/10), .tv_usec = (CRT_delay-((int)(CRT_delay/10)*10)) * 100000 };
+#ifndef timeradd
```

Comment:
I'd probably just hoist this code out in to a `timeradd` function and use `#ifndef timeradd` around its definition.

Reply:
I did try making the macro a function (and/or placing it in a header, but it somehow fouled up the automated header generator.

Reply:
I suppose since this is the only usage in the code, there's not much point separating it out in to a function/macro. My only suggestion then would be to reverse the check to `#ifdef timeradd` - keep the normal path first.